scripts/smoke_test/10-evmserverd.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "=== Checking evmserverd..."
+evmserverd_active=$(sshpass -p $newpassword ssh -tt root@$ip_address "systemctl is-active evmserverd")
+echo "--- Expected: active  Found: ${evmserverd_active}"
+# [[ $evmserverd_active =~ "active" ]] && exit 1 ### TODO: for some reason this is failing string and regex matching
```

Comment:
Could it be a trailing `\r` ?  Wonder if this would work better:

```suggestion
# [[ "$evmserverd_active" =~ "active" ]] && exit 1 ### TODO: for some reason this is failing string and regex matching
```